front/app/components/IdeaForm/index.tsx

```diff
@@ -768,11 +768,16 @@ class IdeaForm extends PureComponent<
         topicsEnabled && allowedTopics && allowedTopics.length > 0;
       const showLocation = locationEnabled;
       const showProposedBudget = proposedBudgetEnabled;
-      const inputTerm = getInputTerm(
-        project.attributes.process_type,
-        project,
-        phases
-      );
+
+      const uiSchemaOptions =
+        ideaCustomFieldsSchemas?.ui_schema_multiloc[locale]?.options;
+
+      const inputTerm = () => {
```

Comment:
We can just make this a variable and not a function. But if we must make it a function, we could rename it to `getInputTerm`